packages/ffe-file-upload-react/src/FileItem.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { number, func, shape, object, string as stringType } from 'prop-types';
+
+const FileItem = props => (
+    <li key={props.index}>
+        {// File is loading
+        !props.file.document && !props.file.error && (
+            <div className="ffe-file-upload__file-item-stencil">
```

Comment:
There are alot of `div`s throughout this component.
I'm no expert on semantics, but could any of these `div`s possibly be something else to add semantics?

@ivarni

Reply:
I'm not familiar with the correct semantics for file uploads but we discussed the accesibility part of it and decided we'd make a separate issue for that. Since the upload button is inside the drag-and-drop target container it's difficult to pull that off without just providing a second upload button that is hidden for non-screenreaders outside of it and then just hiding everything else for screenreaders. At that point semantics matter a bit less.

But I didn't really want to mix that into this redesign since this PR is already quite big.